test/fixtures/complex/quoted_after_matcher.mf2

```diff
@@ -23,6 +23,10 @@ Declarations are not valid after the message body. Did you mean to put the decla
 Message has multiple bodies, but only one is allowed. (at @33..37)
```

Comment:
Can we add a "delete this body" fix on both bodies?

Reply:
Not easially, because the diagnostic is only on one of the two bodies. We'd need to implement diagnostics with more than one range first.